CONTRIBUTING.md

```diff
@@ -27,3 +27,14 @@ and final decision is with the Bitcoin Core project lead.
 
 Translations should be made from the English source. The English source has a `version:` field in the Front Matter which should be updated if on the rare
 occasion the English text is updated. Translated text should remove `translated: false` from the Front Matter.
+
+## License
+
+The code and documentation of this website are licensed under [MIT license][MIT] and all contributors agree to license their
```

Comment:
Is it `documentation of this website` or `documentation on this website (i.e. Bitcoin Core documentation)`?